lib/vrl/compiler/src/expression/assignment.rs

```diff
@@ -3,8 +3,9 @@ use crate::parser::{
     ast::{self, Ident},
     Node,
 };
-use crate::{Context, Expression, Path, Span, State, TypeDef, Value};
```

Comment:
It might have been a good idea to have a `type Path = LookupBuf;`, just to minimize the API changes in VRL.

I still somewhat like that idea, as it makes it a bit more clear what the type represents in VRL's context, but I'm also fine leaving things as they are.

Reply:
I'm on the side of leaving things as they are. I think it makes things clearer when we know exactly what the type is.